hi-ml-azure/testazure/testazure/test_himl.py

```diff
@@ -119,6 +120,103 @@ def test_write_run_recovery_file(mock_run: mock.MagicMock) -> None:
     assert expected_run_recovery_id != recovery_file_text
 
 
+@pytest.fixture(scope="module")
+def dummy_max_num_nodes_available() -> int:
+    return randint(2, 10)
+
+
+@pytest.fixture(scope="module")
+def dummy_compute_cluster_name() -> str:
```

Comment:
I don't get the point of using a constant in a fixture.

Reply:
Because it is shared between multiple tests

Reply:
yes, but why make it a fixture, and not a plain constant in that file?

Reply:
It was just for consistency. Happy to make this a constant if you prefer?